test/jdk/java/awt/print/PrinterJob/PrintLatinCJKTest.java

```diff
@@ -91,6 +102,8 @@ public static void main(String[] args) throws InterruptedException, InvocationTa
                 "Frame", info, 7, 30, 3);
         PrintLatinCJKTest.showFrame();
         passFailJFrame.awaitAndCheck();
+        PrintLatinCJKTest.disposeTestFrame();
```

Comment:
It won't be disposed of if the test fails, that is throws an exception, so this should be in try-finally block.

The class name isn't required to call a static method which is in this class.

Reply:
Fixed